_config.yml

```diff
@@ -16,10 +16,18 @@
 title: Just the Docs
 description: A Jekyll theme for documentation
 baseurl: "/just-the-docs" # the subpath of your site, e.g. /blog
-url: "https://pmarsceill.github.io" # the base hostname & protocol for your site, e.g. http://example.com
+url: "https://pdmosses.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
We probably want to put this back before shipping 😅 

```suggestion
url: "https://pmarsceill.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

Reply:
Oops... indeed, it wasn't intentional 😃